skorch/net.py

```diff
@@ -1010,6 +1039,46 @@ def infer(self, x, **fit_params):
             return self.module_(**x_dict)
         return self.module_(x, **fit_params)
 
+    def get_predict_nonlinearity(self):
```

Comment:
With this being public there will be two ways to adjust the nonlinearity:

1. With the `predict_nonlinearity` parameter in init.
2. Subclassing

Are we okay with this?

Reply:
Good question, I was also unsure about this. The `predict_nonlinearity` parameter specifies _what_ nonlinearity I want, whereas the `get_predict_nonlinearity` specifies _how_ the nonlinearity selection is resolved. I would also be okay with making the latter private (with the possibility to make it public later if the need ever arises).

Reply:
I would go with option 1 for now and having this function be public.

If a user were to override `_get_predict_nonlinearity`, we would not officially support it and it may break in the future.

Reply:
Just to be sure, option 1 for you means making `get_predict_nonlinearity` private?